server/pulp/server/webservices/settings.py

```diff
@@ -10,6 +10,12 @@
 
 TEMPLATE_DEBUG = False
 
+TEMPLATES = [
```

Comment:
I don't think this will work for older versions of Django. The Django docs says this setting was [introduced on 1.8](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-TEMPLATES).

We need to support as far back as 1.4 for EL6:  https://apps.fedoraproject.org/packages/Django14